modules/renter/repair.go

```diff
@@ -227,6 +273,18 @@ func (r *Renter) threadedRepairFile(name string, meta trackedFile) {
 		r.repairChunks(f, handle, badChunks, duration)
 	}
 
+	// repair offline chunks
+	if badChunks := f.offlineChunks(r.hostDB); len(badChunks) != 0 {
```

Comment:
should only be repairing if 1/2 the error margin on redundancy is gone.

So if there are 8 parity pieces, repair when 4+ are missing. This is to help keep churn low, and prevent from continually uploading to crappy hosts.